packages/prisma/src/data/data/TEST.ts

```diff
@@ -1,21 +1,6 @@
 import Prisma from '../../../dist'
 import { LearningElementStatus } from '../../client'
-const { AttachmentType, QuestionType, SessionStatus, OrderType } = Prisma
-
-export const ATTACHMENTS = [
-  {
-    id: 'b0b9c0c0-0b0b-4b4b-0b0b-0b0b0b0b0b0b',
-    href: 'https://sos-ch-dk-2.exo.io/klicker-uzh-dev/avatars/41b98856a8c221db667cf066f34b931eff048c32.svg',
-    name: 'Test Attachment 1',
-    type: AttachmentType.SVG,
-  },
-  {
-    id: 'b0b9c0c0-0b0b-4b4b-0b0b-0b0b0b0b0b0c',
-    href: 'https://sos-ch-dk-2.exo.io/klicker-uzh-dev/avatars/41b98856a8c221db667cf066f34b931eff048c32.webp',
-    name: 'Test Attachment 2',
-    type: AttachmentType.WEBP,
-  },
-]
+const { QuestionType, SessionStatus, OrderType } = Prisma
```

Comment:
Do we still have some example data in one of the buckets, which we could include in a seed so that images are not just left out and might break during development / layout refactors, when not checked manually and explicitely?

Reply:
TBD later together with a cypress test that uploads some files (but will need to clean them from azure afterwards, so probably also some deletion mechanism required)